hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMasterCommandLine.java

```diff
@@ -244,6 +244,7 @@ private int startMaster() {
           throw new RuntimeException("HMaster Aborted");
       }
     } catch (Throwable t) {
+      t.printStackTrace();
```

Comment:
Will the error be present in the log files with this change? Is there a better way to do this, but still using the logger?

Reply:
I thought about that but as per the screenshot provided in the description, the Exception provided is not available in log file because Logger is not yet loaded.
Anyways, adding printstacktrace() would ensure that the StackTrace is atleast available in .out file. If the Logger is created and loaded by that time, then it would also be present in .log file